gtsfm/data_association/point3d_initializer.py

```diff
@@ -21,7 +21,8 @@
 )
 
 import gtsfm.utils.logger as logger_utils
-from gtsfm.common.sfm_track import SfmMeasurement, SfmTrack2d
+from gtsfm.common.sfm_track import SfmTrack2d
+from gtsfm.utils.reprojection import compute_point_reprojection_errors
```

Comment:
lets not import functions from files.

Reply:
maybe`import from gtsfm.utils.reprojection import as reproj_utils`?

Reply:
fixed